great_expectations/core/http.py

```diff
@@ -0,0 +1,60 @@
+import requests
+from requests.adapters import HTTPAdapter, Retry
+
+from great_expectations import __version__
+
+DEFAULT_TIMEOUT = 20
+
+
+class TimeoutHTTPAdapter(HTTPAdapter):
```

Comment:
Do you want to make this private by prepending a `_`?

Reply:
done!